package.json

```diff
@@ -47,6 +47,7 @@
     "truffle-bytecode-manager": "^1.1.1",
     "truffle-extract": "^1.2.1",
     "web3-eth-abi": "1.2.5",
+    "web3-provider-engine": "^15.0.12",
```

Comment:
Do we need to include this one too?

Reply:
For some reason, it was complaining locally, let me try removing it